Package.swift

```diff
@@ -34,6 +34,10 @@ let package = Package(
 			name: "Common",
 			targets: ["Common"]
 		),
+		.library(
+			name: "API",
```

Comment:
Maybe we want to have a more descriptive name here since when importing it is just
```swift
import API
```

WDYT?

Reply:
How about we call it `TidalApi`? That seems to be the easiest way to reduce confusion...

Reply:
I had the same thought 🙂 

Thank you, will do!

Reply:
I wouldn't camel-case the "API" part though 🙂

Reply:
You wouldn't, I would 🤣 
<img width="608" alt="image" src="https://github.com/user-attachments/assets/a0c4290a-2e9d-4e61-bd5a-3a1adfcc5df6">

Reply:
Thank you! I will definitely follow this advice when I have a chance to commit to Java/Kotlin repos 😄 

https://www.swift.org/documentation/api-design-guidelines/#general-conventions 

<img width="852" alt="Screenshot 2024-10-22 at 16 24 38" src="https://github.com/user-attachments/assets/3274f141-4327-490c-9f1f-a4fe1bf4aa03">